website/pages/docs/overlay/tooltip.mdx

```diff
@@ -36,6 +36,26 @@ If the `children` of Tooltip is a `string`, we wrap with in a `span` with
 > Note 🚨: If the `Tooltip` is wrapping a functional component, ensure that the
 > functional component accepts a `ref` using `forwardRef`.
 
+### With a forwardRef
+
+```jsx manual=true
+const CustomCard = React.forwardRef(({ text, ...rest }, ref) => (
+  <Box p="1">
+    <Tag ref={ref} {...rest}>
+      {text}
+    </Tag>
+  </Box>
+))
+
+const CustomToolTip = () => (
+  <Tooltip label="Hover me" fontSize="md">
```

Comment:
- Please remove `fontSize` prop here, it doesn't do anything.
- Consider using `children` instead of `text` prop in the example.
- Change subheading to "Using custom components"